File: MT.Model/Business/Project_AttachDetail.cs

using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region Project_AttachDetail:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a Project_AttachDetail:EntityBase.
    /// </summary>
    public class Project_AttachDetail : EntityBase
    {
        [Key]
        public Guid Project_AttachDetailID { get; set; }
        public Guid ProjectID { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public int SortOrder { get; set; }
        public string Description { get; set; }
        public Guid? FileResourceID { get; set; }
    }
    #endregion
}

File: BECore/app/model/Project_AttachDetail.js
/// <summary>
/// Model EXTJS Project_AttachDetail.
/// </summary>
Ext.define('QLDT.model.Project_AttachDetail', {
    extend: 'Ext.data.Model',

    idProperty: 'Project_AttachDetailID',
    idType: 'Guid',

    fields: [
        { name: "Project_AttachDetailID", type:"string" },
		{ name: "ProjectID", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "IsTemp", type: "boolean" },
        { name: "Description", type: "string" },
        { name: "FileResourceID", type: "string" },
        
    ]
});

